rails-homework/app/views/physicians/new.html.erb

```diff
@@ -0,0 +1,4 @@
+<div class="container">
+  <h1>Add new physician</h1>
+  <%= render "form", physician: @physician %>
```

Comment:
You do not need `physician: @physician` here. `@physician` will be accessible in partial form.